src/__tests__/index.test.js

```diff
@@ -227,7 +266,10 @@ describe("ServerlessClientBuildPlugin tests", () => {
 
         expect(childProcess.spawn).toHaveBeenCalledWith(
           constants.packagers[packager],
-          ["build"]
+          ["build"],
+          {
+            cwd: undefined
+          }
```

Comment:
You shouldn't need to include `cwd: undefined` in this object. I think just `{}` should work. Same comment for the other instances below

Reply:
The test fails if you remove this.
The object gets passed with a cwd option that is set as undefined so you need to check for this. It is not not an empty object.